examples/compiled/area.vg.json

```diff
@@ -122,7 +122,7 @@
             "tickCount": {
                 "signal": "ceil(width/40)"
             },
-            "title": "Year-month of date",
+            "title": "date (year-month)",
```

Comment:
I think this is worse for most examples like

`date (month)` or `date (date)` instead of `month of date` or `date of date`.

Reply:
I actually think the opposite way that's why I make the change.

For aggregate, I think having "Sum/mean/median of" is critical information so it should not be in parentheses. 
Meanwhile, for time unit, we often see people just shows time data without specifying units.  
Thus, putting time unit in parentheses might better signify that they are units that we display.

I don't see why `month of date` is better than `date (month)`.  Meanwhile, the confusing `year of year` definitely happens more than `date of date`.  Doing `year (year)` signifies that the `year` in parentheses is a unit.  I agree that `date (date)` is confusing, but it should rarely happen and`date of date` is not really less confusing anyway.

Reply:
Okay, I don't have strong opinions here. Let's merge it.